src/helpers/coolParameters.js

```diff
@@ -46,9 +46,9 @@ const getCollaboraTheme = () => {
 const generateCSSVarTokens = () => {
 	/* NC versus COOL */
 	const cssVarMap = {
-		'--color-primary-text': '--co-primary-text',
+		'--color-primary-element-text': '--co-primary-text',
 		'--color-primary-element': '--co-primary-element:--co-text-accent',
-		'--color-primary-light': '--co-primary-light',
+		'--color-primary-element-light': '--co-primary-light',
```

Comment:
not sure about these

Reply:
Will need to double check where they are actually used within Collabora

Reply:
Did you already check with them? :)

Reply:
Not yet, but with 27 and Collabora 25.05 we actually pass over the values directly and make use of the element ones mostly in the branding directly. Still on my todo list for the older versions.